.circleci/config.yml

```diff
@@ -690,12 +690,12 @@ jobs:
             # export env from file (created in build-playwright-job)
             cat playwright-env/envvars | awk '{print "export " $0}' >> "$BASH_ENV"
             source $BASH_ENV
-      - node/install:
-          install-yarn: false
-          node-version: '18.13.0' # LTS version
```

Comment:
Install custom nodejs version is not working again. It has been very flaky lately. Fallback to default nodejs version provided by CircleCI default.

Reply:
Latest failure: https://app.circleci.com/pipelines/github/broadinstitute/ddp-angular/26016/workflows/b87b06cc-58ed-4bcb-a9b7-d4f85d4d6e96/jobs/84566